src/map/skill.cpp

```diff
@@ -11812,8 +11812,16 @@ TIMER_FUNC(skill_castend_id){
 		//These should become skill_castend_pos
 		switch (ud->skill_id) {
 			case WE_CALLPARTNER:
-				if (sd)
-					clif_callpartner(sd);
+				if (sd) {
+					struct map_session_data *p_sd = pc_get_partner(sd);
+					if (!p_sd || p_sd->state.autotrade) {
```

Comment:
Compared to WE_CALLPARENT and WE_CALLBABY we now fail the skill even if the partner is not online.
In the other two skills we only fail if the other player is online and in autotrade state.
This might explain why clif_callpartner originally supported calls without a target player session too.